CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ different versioning scheme, following the Haskell community's
 * Fixed MSVC warning for deprecation of `std::result_of_t` in `/std:c++17`.
   ([Issue #1007](https://github.com/microsoft/bond/issues/1007))
 
+### C# ###
+
+* Fixed creation of duplicate default constructor when defining an empty struct and using --contructor-parameters.
```

Comment:
Minor: \` backticks \` around `--constructor-parameters`?
Minor: typo in <code>--con<strong>s</strong>tructor-parameters</code>
Minor: line is a bit long.
Minor: isn't the switch `--preview-constructor-parameters`?

```suggestion
* Fixed creation of duplicate default constructor when defining an empty
  struct and using `--preview-constructor-parameters`.
```